scripts.js

```diff
@@ -33,10 +33,24 @@ option3.addEventListener('click', () => {
   header.style.display = 'block';
 });
 
+const msg = document.querySelector('.input-textarea');
+const userName = document.querySelector('.input-name');
 const form = document.getElementById('form');
 const email = document.getElementById('email');
 const error = document.querySelector('.error-msg');
 
+function storeData() {
+  const info = {
+    name: userName.value,
+    email: email.value,
+    message: msg.value,
+  };
+  window.localStorage.setItem('info', JSON.stringify(info));
+}
+
+form.addEventListener('submit', storeData);
```

Comment:
- [ ] Great job working on saving the data to the localStorage. :+1: Currently, it is saving the data when the form is submitted. But we're supposed to save the data when the user is inputting data in the text fields. Kindly implement it in such a way as to achieve the project requirements. ✍️

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/225429238-2c9aaec9-6c74-47ab-b583-926afdede554.png)